### DISCUSSION TITLE
Is there a way I can categorize the sites?

### DISCUSSION THREAD

**Discussion by vcidst** [upptime/upptime#145] [Q&A] [answered]
Currently all the sites appear in a single list. Is there a way to categorize or group them so that they appear in different sections?

Context: I'm [trying](https://verloop.github.io/status/) to add a lot of our dev services. The list will soon be more than 20 sites, some kind of grouping will allow being able to see the status with a glance. (Or perhaps the decreasing the height of each row will also serve the purpose?)

**Comment by AnandChowdhary** [comment, marked as answer]
Closing as a duplicate of https://github.com/upptime/upptime/discussions/121.
